### PR TITLE
(MAINT) Update content; style changes

### PR DESCRIPTION
**Pull request by petergmurphy**
Overview of PR:

- Fixed padding on cards for small screens.
- Added README instructions on how to deploy the site to GH pages.
- Removed porridge image that appeared in Slack previews of the site.
- Removed redundant layouts.
- Added the blog page section to the site by taking it out of draft. Also, added a blog post.
- Fixed broken links on homepage.
- Change footer colour to the same as the navbar.
- Updated footer copyright information.

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      </div>
+      <div class="col-6 col-sm-4 text-right text-xs-center order-sm-3">
+        {{ with $links }}
+        {{ with index . "developer"}}
+        {{ template "footer-links-block"  . }}
+        {{ end }}
+        {{ end }}
+      </div>
+      <div class="col-12 col-sm-4 text-center py-2 order-sm-2">
+        {{ with .Site.Params.copyright }}<small class="text-white">&copy; {{ now.Year}} {{ .}} {{ T "footer_all_rights_reserved" }}</small>{{ end }}
```
Comment (da-ar) [inline layouts/partials/footer.html:20]: update copyright in config.toml to "Puppet"

**Review by da-ar — CHANGES_REQUESTED** [review]
minor change
Screenshot 2021-12-15 123602

File: assets/scss/_styles_project.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+footer {
+    background-color: $primary;
+}
```
Comment (michaeltlombardi) [inline assets/scss/_styles_project.scss:3]: nit: files should all have final newlines

File: layouts/partials/footer.html (hunk 1 above)
Comment (sanfrancrisko) [reply layouts/partials/footer.html:20]: Apologies - my bad, I said that it was probably OK to do that, following on from the VSCode site convention

Comment (petergmurphy) [reply layouts/partials/footer.html:20]: Updated the `config.toml` `copyright` variable to "Puppet".

**Review by michaeltlombardi — APPROVED** [review]
Built locally and looked it over, seems rad! Not a problem to fix _here_ but I noticed that the PRM changelog shows up in the site while the PCT changelog does not, I think we at least want to make sure those get pulled over consistently.

One problem I did notice is that while the `About` button link on the home page leads to the blog, the `About` entry in the navbar just leads to a blank page. I think that page has always been blank so we don't need to block this changeset on it, but that does need to be addressed as well.

**Comment by petergmurphy** [comment]
> Built locally and looked it over, seems rad! Not a problem to fix _here_ but I noticed that the PRM changelog shows up in the site while the PCT changelog does not, I think we at least want to make sure those get pulled over consistently.
> 
> One problem I did notice is that while the `About` button link on the home page leads to the blog, the `About` entry in the navbar just leads to a blank page. I think that page has always been blank so we don't need to block this changeset on it, but that does need to be addressed as well.

So running the site with the `hugo server` command will only show the pages not in draft, as the `about` page is still in draft I decided to just link the `about` homepage button to the blog post, just so it leads somewhere.

File: README.md [hunk 3]
```diff
@@ -31,3 +31,8 @@ When using the `get` command, if the `cannot find module providing package` mess
 
 `hugo mod clean --all`
 
+## Updating the live site
+
+The site will be deployed when commits are merged into the main branch of the repo.
+
+Alternatively, the site can be deployed by running the `github_pages` workflow action in the Github Actions section.
```
Comment (sanfrancrisko) [inline README.md:38]:
```suggestion
Alternatively, the site can be deployed by running the [`github_pages` workflow action](https://github.com/puppetlabs/devx/actions/workflows/gh-pages.yml).
```

**Comment by petergmurphy** [comment]
> Built locally and looked it over, seems rad! Not a problem to fix _here_ but I noticed that the PRM changelog shows up in the site while the PCT changelog does not, I think we at least want to make sure those get pulled over consistently.

I think we'll need to look into how we want to display the changelog/versions on the site as currently the PRM `CHANGELOG.md` that's being displayed is just an a copy of the `CHANGELOG.md` from the project root dir.

**Pull request merged**